dbos/_migrations/versions/258053ffda6c_set_default_value_of_recovery_attempts_.py

```diff
@@ -0,0 +1,36 @@
+"""Set default value of recovery_attempts to 1
+
```

Comment:
Why is this needed? Why can't the default be 0?

Reply:
we don't need it and actually wouldn't have work with our enqueuing logic